src/ResourceManager/Resources/Commands.Resources/Models.ResourceGroups/ResourceClient.cs

```diff
@@ -325,16 +325,17 @@ public static List<string> ParseDetailErrorMessage(string statusMessage)
             params string[] status)
         {
             DeploymentExtended deployment;
+            Delay stepedDelay = new Delay(firstDelay: 0);
 
             do
             {
+                TestMockSupport.Delay(stepedDelay.GetNextDelay());
```

Comment:
Per our sync up, will change it to the simple way as we don't see future reuse of this.